pkg/apis/operator/v1alpha1/openshift_platform.go

```diff
@@ -30,3 +33,15 @@ type PipelinesAsCode struct {
 	// +optional
 	PACSettings `json:",inline"`
 }
+
+type SCC struct {
+	// Default contains the default SCC that will be attached to the service
+	// account used for workloads (`pipeline` SA by default) and defined in
+	// PipelineProperties.OptionalPipelineProperties.DefaultServiceAccount
+	// +optional
+	Default string `json:"default,omitempty"`
+	// MaxRequest specifies the highest SCC that can be requested for in a
+	// namespace or in the Default field.
+	// +optional
+	MaxRequest string `json:"maxRequest,omitempty"`
```

Comment:
we discussed to change this field, can we please do that change

Reply:
changed to `maxAllowed`